JSDemos/Demos/Charts/Spline/description.md

```diff
@@ -1 +1,7 @@
-Spline series types, like line series types, help visualize data trends over specified intervals. However, splines help &laquo;smooth&raquo; individual lines that connect data points. In&nbsp;this example, you can switch between the _spline_, _stacked spline_ and _full-stacked spline_ series types. _Stacked spline_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked spline_ series help compare the percentage value of&nbsp;multiple spline series for each argument.
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Spline/jQuery/Light/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
```

Comment:
Remove the suffix that specifies the framework and the theme. Our website stores the selected framework and theme in cookies and redirects the user to the corresponding version of a demo based on this information. If you keep the suffix, your link will point at a specific framework and a specific theme, which is rarely needed.

```suggestion
Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Line/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
```